opencti-platform/opencti-front/src/private/components/arsenal/vulnerabilities/VulnerabilityCreation.tsx

```diff
@@ -126,9 +126,9 @@ export const VulnerabilityCreationForm: FunctionComponent<VulnerabilityFormProps
       createdBy: values.createdBy?.value,
       objectMarking: values.objectMarking.map((v) => v.value),
       objectLabel: values.objectLabel.map((v) => v.value),
-      x_opencti_base_score: parseInt(String(values.x_opencti_base_score), 10),
-      x_opencti_base_severity: values.x_opencti_base_severity,
-      x_opencti_attack_vector: values.x_opencti_attack_vector,
+      x_opencti_cvss_base_score: parseInt(String(values.x_opencti_cvss_base_score), 10),
```

Comment:
This part of the code forces the score to be an integer, not a float:

![test-indexing-cve-score-float](https://github.com/OpenCTI-Platform/opencti/assets/75518128/1f06269e-b2da-4e42-8f9d-a580abe2de6a)

I suggest to add parseFloat() instead:
```suggestion
      x_opencti_cvss_base_score: parseFloat(Number(values.x_opencti_cvss_base_score).toFixed(2)),
```

Then, when create a new CVE, we have the float number:

![test-indexing-cve-score-float-2](https://github.com/OpenCTI-Platform/opencti/assets/75518128/959a53d3-ae58-47ef-9cff-bbe6934333ea)